backend/server/server.js

```diff
@@ -1,29 +1,92 @@
-const mockedData = require("../../frontEnd/src/components/data.json");
 const express = require("express");
 const app = express();
-const port = 1003;
+const port = 2003;
+require("dotenv").config();
+const cors = require("cors");
+const multer = require("multer");
+const upload = multer({ storage: multer.memoryStorage() });
+const axios = require("axios");
+const FormData = require("form-data");
+const mockedData = require("../../frontEnd/src/components/data.json");
+const mongoose = require("mongoose");
+const Hackathon = require("./HackathonModel");
+const apiKey = process.env.API_KEY;
+app.use(cors());
+console.log(process.env.MONGODB_STR);
```

Comment:
Console log can be removed, this can expose your mongodb connection string on server